index.js

```diff
@@ -1,102 +1,106 @@
 /**
- * Винести змінні які ти мутуєш в окремий об'єкт як ключі
- * Пройтись по змінних і замінити let на const у місцях де це можливо
- * Змінні в camelCase
- * DRY principle: накнайменше дублікацій коду
- * Переписати проміси на async/await
+ * Винести змінні які ти мутуєш в окремий об'єкт як ключі +
+ * Пройтись по змінних і замінити let на const у місцях де це можливо +
+ * Змінні в camelCase +
+ * DRY principle: накнайменше дублікацій коду +
+ * Переписати проміси на async/await +
  * */
 
 
 (function (){
   const pagination = {
-
+    page: 1,
+    perPage: 2,
+    total: document.getElementById('total'),
+    totalCount: 0,
+    pageNumberSpan: document.getElementById('page-number'),
```

Comment:
Do we need `pageNumber` property? Cause we already have `page` prop? What is the difference between them?